input.go

```diff
@@ -188,26 +188,35 @@ func (t *Terminal) TypedShortcut(s fyne.Shortcut) {
 	}
 
 	if runtime.GOOS == "darwin" {
-		// do the default thing for macOS as they separete ctrl/cmd
+		// do the default thing for macOS as they separate ctrl/cmd
 		t.ShortcutHandler.TypedShortcut(s)
 	} else {
 		// we need to override the default ctrl-X/C/V/A for non-mac and do it ourselves
 
-		if _, ok := s.(*fyne.ShortcutCut); ok {
-			_, _ = t.in.Write([]byte{0x18})
-
-		} else if _, ok := s.(*fyne.ShortcutCopy); ok {
-			_, _ = t.in.Write([]byte{0x3})
-
-		} else if _, ok := s.(*fyne.ShortcutPaste); ok {
-			_, _ = t.in.Write([]byte{0x16})
-
+		if ct, ok := s.(*fyne.ShortcutCut); ok {
+			if ct.Secondary {
```

Comment:
Perhaps you could have exposed the triggering keys/key-code on the shortcut struct along side the `Secondary` bool? This would fix your issue here of trying to determine what keys to translate them into here; you could just write whatever key-code was originally captured when converting the input into a shortcut. The added bonus being that developers like my self would then be able to tell the exact keys used to trigger the shortcut and add custom handling if we choose to do so.